Components/LearningCore/Source/gov/sandia/cognition/learning/algorithm/clustering/OptimizedAffinityPropagation.java

```diff
@@ -0,0 +1,261 @@
+package gov.sandia.cognition.learning.algorithm.clustering;

```

Comment:
Hmmm....  Whitespace is different with this file versus all the other files in the Foundry.  (This has 2-space tabs generally, while the rest of the Foundry uses 4-space tabs.)  Could you please fix this?

Reply:
I hope to have done